sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/WebExceptionRetryPolicy.java

```diff
@@ -23,7 +26,8 @@ public class WebExceptionRetryPolicy implements IRetryPolicy {
     private HttpTimeoutPolicy timeoutPolicy;
     private boolean isReadRequest;
     private int retryCount = 0;
-    private URI locationEndpoint;
+    private RegionalRoutingContext regionalRoutingContext;
+    private URI overriddenEndpoint;
```

Comment:
what is this overridenEndpoint? what will be the value of this?

Reply:
Our `Read` of `DatabaseAccount` path uses an `overridenEndpoint` and not the `locationEndpointToRoute`. `WebExceptionRetryPolicy` only recognized `locationEndpointToRoute` yet is used by `DatabaseAccount` reads so made that accommodation here.